### PR TITLE
menu above buttons resolve

### PR DESCRIPTION
**Pull request by mementomori16**

### CODE REVIEW

File: src/view/components/menu/Menu.scss [hunk 1]
```diff
@@ -6,7 +6,7 @@
         flex-direction: column;
         background-color: white;
         box-shadow: 0px 3px 9px #00000026;
-        z-index: 5;
+        z-index: 102;
```
Comment (vladbyk89) [inline src/view/components/menu/Menu.scss:9]: I don't like this solution. Can you lower the z-index on the button instead?

File: .firebaserc [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+  "projects": {
+    "default": "delib-5-72f2d",
+    "dev": "delib-v3-dev",
+    "prod": "synthesistalyaron"
+  },
+  "targets": {},
+  "etags": {}
+}
```
Comment (vladbyk89) [inline .firebaserc:9]: Do not commit this file

File: firebase.json [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    },
+    "singleProjectMode": true,
+    "hosting": {
+      "port": 5000
+    }
+  },
+  "storage": {
+    "rules": "storage.rules"
+  }
+}
```
Comment (vladbyk89) [inline firebase.json:61]: Do not commit this file

**Review by vladbyk89 — CHANGES_REQUESTED** [review]

File: .firebaserc (hunk 2 above)
Comment (mementomori16) [reply .firebaserc:9]: it was a mistake

File: firebase.json (hunk 3 above)
Comment (mementomori16) [reply firebase.json:61]: it was a mistake

File: src/view/components/menu/Menu.scss (hunk 1 above)
Comment (mementomori16) [reply src/view/components/menu/Menu.scss:9]: Yes, I lowered the z-index on the button instead

**Pull request merged**